core-bundle/tests/Contao/ImageTest.php

```diff
@@ -1001,8 +1001,6 @@ public function getGetLegacy(): \Generator
     public function testDoesNotFactorImagesInTheLegacyMethodIfTheArgumentIsInvalid(): void
     {
         $this->assertNull(Image::get('', 100, 100));
-        $this->assertNull(Image::get(0, 100, 100));
-        $this->assertNull(Image::get(null, 100, 100));
```

Comment:
Why are these assertions no longer needed?

Reply:
Because they test with invalidly typed arguments.

Reply:
These tests should make sure that we keep BC, I would like to keep them. @m-vo could this stay if we configure phpstan to mistrust the function parameter annotated types?

Reply:
Had a quick discussion with @ausi about this topic. Probably PHPStan should ignore these cases where the type checking is done inside the method for legacy reasons (no real types in function signature + no `strict_types=1`) but it treats the annotations as certain.

I tried `treatPhpDocTypesAsCertain: false` but unfortunately this does not affect function parameters, so I'm going to ignore the errors for these few occurrences in our `phpstan.neon.dist` config.

Reply:
Handled in daf0e0407d50b06b150e4997865c4629c4cdc71d.